addons/web/static/tests/core/dropdown_tests.js

```diff
@@ -50,7 +50,7 @@ QUnit.module("Components", ({ beforeEach }) => {
         await mount(Parent, target, { env });
         assert.strictEqual(
             target.querySelector(".dropdown").outerHTML,
-            '<div class="o-dropdown dropdown o-dropdown--no-caret"><button class="dropdown-toggle"></button></div>'
+            '<div class="o-dropdown dropdown o-dropdown--no-caret"><button class="dropdown-toggle" tabindex="0"></button></div>'
```

Comment:
add small test asserting the new prop changes that

Reply:
or maybe not, as you tested it in one2many_field_tests

Reply:
Is it okay if I don't write the test?

Reply:
ok for me